web/models.py

```
@@ -312,6 +312,11 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+    def delete(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `delete`

Add return type annotation: `None`

(ANN201)